patientsearch/src/js/components/PatientListTable.js

```diff
@@ -736,14 +760,23 @@ export default function PatientListTable() {
             .map((item) => item.resource.id)
             .join(",");
           // FHIR resources request(s)
-          const requests = additionalParams.map((queryString) =>
-            fetchData(
-              `/fhir/${queryString}` +
-                (queryString.indexOf("?") !== -1 ? "&" : "?") +
-                `patient=${ids}&_count=1000`,
-              constants.noCacheParam
-            )
-          );
+          const requests = eligibleRequests.map((request) => {
+            let queryString = "";
+            let { resourceType, queryParams, referenceElement } = request;
+            if (!referenceElement) referenceElement = "patient";
+            let params = ["_count=1000", `${referenceElement}=${ids}`];
```

Comment:
do you recall why we include `_count=1000` here?  i have a vague memory of that being a workaround prior to having HAPI pagination plugged in on COSRI?